ts/features/it-wallet/utils/mocks.ts

```diff
@@ -37,9 +36,18 @@ export const mapAssuranceLevel = (level: AssuranceLevel | string) => {
 
 export const CREDENTIAL_ISSUER = "eFarma";
 
+/**
+ * Credential types mocks.
+ */
+export enum CredentialType {
+  HEALTH_CARD = "healthCard",
+  EUROPEAN_DISABILITY_CARD = "EuropeanDisabilityCard",
```

Comment:
We don't have control over this. Even though this is a mocked value (it should be received from a remote list), it's being used to filter the issuer configuration. 
The other entries in camelCase are not updated with the new casing because they are not being used and their value is not known at the moment. However I updated them to be consistent.